### PR TITLE
Bootstrap And Typos

### LINKED ISSUE #15: Small screen supported site
- fix disappearing text from each section
- fix font size inconsistencies.

### PR DESCRIPTION
**Pull request by MortarDefender**
- bootstrap added to support smaller screens and to beautify the site
- index.html and index_style.css has fixed picture face cutting in the
  meet the team section

fix: #15

### CODE REVIEW

**Comment by brochwerger** [comment]
README.md still has typos. Please review all English files

**Review by brochwerger — CHANGES_REQUESTED** [review]
Looks good but I noticed at least a couple of typos in README.md that haven't been fixed 
Architectue ? Datebase ?

Also, I think it is cleaner to separate the fixes of typos from styling changes (in .css) files.

**Comment by MortarDefender** [comment]
Done. 
This PR is only for style change on the main site.
See a new PR for typos  #17

**Comment by MortarDefender** [comment]
@YaelD @RavidYael @danninon @BelikovYoav @RonHachmon
can you please write a code review on this pull request,
see what has changed, and if there is a need to add or modify anything.
Thanks

**Review by danninon — DISMISSED** [review]

File: main/static/main/index_js.js [hunk 1]
```diff
@@ -44,13 +44,12 @@ function present(name, image_url, title, attributes) {
...
-           "I'm a 3rd year Computer Science Student <br> and a software developer", ["I did a few projects similer to what is required in our course", "I did a few other courses that required team work."])
+           "I'm a 3rd year Computer Science Student <br> and a software developer", ["I did a few projects similar to what is required in our course.", "I did a few other courses that required teamwork."])
 }
 
 function presentYael() {
     present("Yael Davidov", "https://images.unsplash.com/photo-1608534430161-593a93055096?ixlib=rb-1.2.1&ixid=MnwxMjA3fDB8MHxwaG90by1wYWdlfHx8fGVufDB8fHx8&auto=format&fit=crop&w=874&q=80", 
-           "I'm a 4rd year Computer Science Student", ["I did a few courses that are relavent to the work we are going to do such as" +
-                     "java script and computer comunications"])
+           "I'm a 4rd year Computer Science Student", ["I did a few courses that are relevant to the work we are going to do such as javascript and computer communications"])
```
Comment (nemerna) [inline main/static/main/index_js.js]: please change 4rd to 4th

**Review by nemerna — DISMISSED** [review]

**Review by brochwerger — DISMISSED** [review]

File: main/static/main/index_js.js (hunk 1 above)
Comment (nemerna) [reply main/static/main/index_js.js]: You can also use this online tool to improve your writing.
Although it's not a necessity, I sometimes use it when I need to write official mail/requests.
https://www.wordtune.com/

Comment (MortarDefender) [reply main/static/main/index_js.js]: done

**Review by nemerna — APPROVED** [review]
Great!, i cant see any other typos/issues

**Pull request merged**